src/main/java/net/earthcomputer/clientcommands/command/KitCommand.java

```
@@ -0,0 +1,182 @@
+                        .then(argument("name", StringArgumentType.string())
+                                .executes(ctx -> create(ctx.getSource(), StringArgumentType.getString(ctx, "name")))))
+                .then(literal("delete")
+                        .then(argument("name", StringArgumentType.string())
+                                .suggests((ctx, builder) -> CommandSource.suggestMatching(kits.keySet(), builder))
+                                .executes(ctx -> delete(ctx.getSource(), StringArgumentType.getString(ctx, "name")))))
+                .then(literal("edit")
+                        .then(argument("name", StringArgumentType.string())
+                                .suggests((ctx, builder) -> CommandSource.suggestMatching(kits.keySet(), builder))
+                                .executes(ctx -> edit(ctx.getSource(), StringArgumentType.getString(ctx, "name")))))
+                .then(literal("kit")
+                        .then(argument("name", StringArgumentType.string())
+                                .suggests((ctx, builder) -> CommandSource.suggestMatching(kits.keySet(), builder))
+                                .then(literal("--override")
+                                        .executes(ctx -> kit(ctx.getSource(), StringArgumentType.getString(ctx, "name"), true)))
+                                .executes(ctx -> kit(ctx.getSource(), StringArgumentType.getString(ctx, "name"), false))))
+                .then(literal("kits")
+                        .executes(ctx -> kits(ctx.getSource()))));
+    }
+
+    private static int create(ServerCommandSource source, String name) throws CommandSyntaxException {
+        loadFile();
+
+        if (kits.put(name, client.player.inventory) == null) {
+            saveFile();
+            sendFeedback("commands.ckit.create.success", name);
+        } else {
+            saveFile();
+            sendFeedback("commands.ckit.create.alreadyExists", name);
+        }
```

The main point of having the split between edit and create is to protect against people accidentally overwriting kits. So this function should throw an error if the kit already exists.